lightly/transforms/rfft2d_transform.py

```diff
@@ -0,0 +1,42 @@
+from typing import Union
+
+import torch
+import torchvision.transforms as transforms
+from PIL import Image
+from torch import Tensor
+
+
+class RFFT2DTransform:
+    """2D Fast Fourier Transform (RFFT2D) Transformation.
+
+    This transformation applies the 2D Fast Fourier Transform (RFFT2D)
+    to an image, converting it from the spatial domain to the frequency domain.
+
+    Input:
+        - Tensor of shape (C, H, W), where C is the number of channels.
+        - PIL.Image can also be passed and will be converted to a Tensor internally.
+
+    Output:
+        - Tensor of shape (C, H, W) in the frequency domain, where C is the number of channels.
+    """
+
+    def __init__(self):
+        self.to_tensor = transforms.ToTensor()
+        self.to_pil = transforms.ToPILImage()
```

Comment:
We shouldn't add `to_tensor` and `to_pil` here. We can expect that images are already converted to tensors before being passed to `RFFT2D`. I think you can completely remove the `__init__` method.

```suggestion
```